README.md

```diff
@@ -1,77 +1,64 @@
+|  (advanced types, multi-block data structures)   |
+|                                                  |
+├──────────────────────────────────────────────────┤
+|                                                  |
+|                 Data Model Layer                 |
+|   (basic types, single-block data structures)    |
+|                                                  |
+├──────────────────────────────────────────────────┤
+|                                                  |
+|                   Block Layer                    |
+|               (cid, data, codec)                 |           
+|                                                  |
+└──────────────────────────────────────────────────┘
 ```
 
-The goal of this stack is to enable decentralized data-structures
-which in turn will enable more decentralized applications.
+## Block Layer (Layer 0)
 
-Many of the specifications in this stack are inter-dependent.
+The block layer encompasses all content addressed block formats and specifies how blocks
+are addressed,how they self-describe their codec for encoding/decoding, and how block link
+between each other.
 
-```
-                          IPLD Dependency Graph
-
-┌─────┐                           ┌────────────┐
-│ CID ├───────────┬───────────────> Raw Blocks │
-└─────┘           │               └─┬──────────┘
-          ┌───────v──────────────┐  │
-┌──────┐  │ Links (Conceptually) │  │
-│ Path │  └───────┬──────────────┘  │           ┌─────────────┐
-└──┬───┘          │                 ├───────────> Replication │
-   │              │                 │           └─────────────┘
-┌──v──────────────v─────────────────┴──────┐
-│                  Codecs                  │
-│  ┌─────┐   ┌───────────────────────────┐ │ ┌────────────────────────────┐
-│  │ Git │   │       Data Model v1       │ │ │  Multi-block Collections   │
-│  └─────┘   │ ┌──────────┐ ┌──────────┐ │ │ │ ┌────────────────────────┐ │
-│ ┌────────┐ │ │ dag-json │ │ dag-cbor │ ├─┼─> │    Specialized Data    │ │
-│ │  BTC   │ │ └──────────┘ └──────────┘ │ │ │ │ Structures & Utilities │ │
-│ └────────┘ └───────────────────────────┘ │ │ └────────────────────────┘ │
-│ ┌────────┐ ┌────────┐                    │ └───┬─────────┬──────────────┘
-│ │ Zcash  │ │ dag-pb │                    │     │         │
-│ └────────┘ └────────┘                    │     │         │
-└──────────────────────────────────────────┘     │         │
-                                                 │         │
-               ┌──────────────────────┐          │         │
-               │ File System (UnixFS) <──────────┘         │
-               └──────────────────────┘                    │
-               ┌───────────────────────┐                   │
-               │ End-User Applications <───────────────────┘
-               └───────────────────────┘
+This layer alone is enough to accomplish basic graph replication across a variety of formats (eth, bitcoin, git, etc).
```

Comment:
I'm unsure if we should mention formats here. As those certainly live in Layer 1. They just don't implement the full IPLD Data Model (so not everything IPLD can be encoded in them), but all types they have should be decoded in an IPLD Data Model compatible way.

Reply:
maybe we should just add mention of dag-pb and dag-cbor to make it clear that you can do replication with only this layer.